curator/src/run_cron.py

```diff
@@ -18,23 +20,62 @@ def __init__(self):
         self.minute = self.defaults.get('runminute', 0)
         self.timezone = self.defaults.get('timezone', 'UTC')
         self.job_list = CronTab()
+        self.ca = os.getenv('ES_CA', '/etc/curator/keys/ca')
+        self.cert = os.getenv('ES_CLIENT_CERT', '/etc/curator/keys/cert')
+        self.key = os.getenv('ES_CLIENT_KEY', '/etc/curator/keys/key')
+        self.es_url = "https://" \
+            + os.getenv('ES_HOST', 'logging-es') + ':' \
+            + os.getenv('ES_PORT', '9200')
 
     def setup_cron(self):
         for cmd in self.cmd_list:
             job = self.job_list.new(command=cmd, comment='Generated job based on settings')
             job.every().day()
 
+    def server_ready(self):
+        retry_max = 11  # (2**(11+1)-1) / 60 = 68 minutes of exponential back-off
```

Comment:
I think this should be unlimited - curator should keep trying to connect indefinitely